plasma-mvp/plasma-mvp-spec.ini

```diff
@@ -289,6 +289,32 @@ requires:
     andBool #rangeAddress(EXIT_TOKEN)
     andBool #rangeUInt(256, EXIT_AMOUNT)
 
+[getNextExit]
+k: #execute => #halt
+statusCode: _ => EVMC_SUCCESS
+output: _ => intList2ByteStack(UTXO_POS EXITABLE_AT)
+callData: #abiCallData("getNextExit", #address(TOKEN_ID))
+callValue: 0
+gas: {GASCAP} => _
+log: _
+refund: _
+balance: _
+storage:
+    #hashedLocation({COMPILER}, {_EXITSQUEUES}, TOKEN_ID) |-> EXITS_QUEUE_ID
```

Comment:
How about looking up `PRIORITY_QUEUE_ID` directly?

Reply:
Yes, just to clarify I use `EXITS_QUEUE_ID` requirement.
To use `PRIORITY_QUEUE_ID` directly is simpler one.